packages/terra-functional-testing/src/reporters/spec-reporter/wdio-spec-reporter.js

```diff
@@ -2,16 +2,44 @@ const fs = require('fs-extra');
 const path = require('path');
 const WDIOReporter = require('@wdio/reporter').default;
 const getOutputDir = require('./get-output-dir');
+const { eventEmitter } = require('../../commands/utils');
 
 class SpecReporter extends WDIOReporter {
+  constructor(options) {
+    super(options);
+    this.screenshotPaths = [];
+    this.screenshotMap = {};
+
+    // Listen to this event when a screenshot is being captured.
+    eventEmitter.on('terra-functional-testing:capture-screenshot', (latestPath) => {
```

Comment:
these events are only used in the node runtime right? have we looked at using nodes built in event emitter instead of pulling in the eventemitter3 package? https://nodejs.org/docs/latest-v12.x/api/events.html#events_class_eventemitter

Reply:
Yes, it would be better to use Node's native eventEmitter. [TerraWDIOTestDetailsReporter](https://github.com/cerner/terra-toolkit-boneyard/blob/87db811f811d52b4ec46f1917b00a35cb5b1e465/reporters/wdio/TerraWDIOTestDetailsReporter.js#L1) also uses the built in event emitter and I thought this was a third party event emitter.

https://github.com/cerner/terra-toolkit/pull/672/commits/c21b0c3e0465365a38b33b3b7b703c1d0f302943